composite/proof-of-reserves/.prettierrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  ...require('../../.prettierrc.js'),
+}
```

Comment:
```suggestion
module.exports = require('../../.prettierrc.js')
```

Reply:
Doesnt prettier resolve by crawling up the directory tree anyway?

Reply:
I guess we don't 👍